From 422ca0551d119315cb50c98e8f94d2c0a5f435e8 Mon Sep 17 00:00:00 2001
From: Bryan Cantrill <bryan@joyent.com>
Date: Fri, 1 Mar 2019 01:24:21 +0000
Subject: [PATCH] OS-7631 mdb tests assume advanced "case-sensitive" filesystem
 technology Reviewed by: Jordan Hendricks <jordan.hendricks@joyent.com>
 Approved by: Robert Mustacchi <rm@joyent.com>

---
 .../mdb/test/format/{tst.format-E.mdb => tst.format-cap-E.mdb}    | 0
 .../format/{tst.format-E.mdb.out => tst.format-cap-E.mdb.out}     | 0
 .../mdb/test/format/{tst.format-G.mdb => tst.format-cap-G.mdb}    | 0
 .../format/{tst.format-G.mdb.out => tst.format-cap-G.mdb.out}     | 0
 .../mdb/test/format/{tst.format-J.mdb => tst.format-cap-J.mdb}    | 0
 .../format/{tst.format-J.mdb.out => tst.format-cap-J.mdb.out}     | 0
 .../mdb/test/format/{tst.format-P.mdb => tst.format-cap-P.mdb}    | 0
 .../format/{tst.format-P.mdb.out => tst.format-cap-P.mdb.out}     | 0
 .../mdb/test/format/{tst.format-R.mdb => tst.format-cap-R.mdb}    | 0
 .../format/{tst.format-R.mdb.out => tst.format-cap-R.mdb.out}     | 0
 10 files changed, 0 insertions(+), 0 deletions(-)
 rename usr/src/cmd/mdb/test/format/{tst.format-E.mdb => tst.format-cap-E.mdb} (100%)
 rename usr/src/cmd/mdb/test/format/{tst.format-E.mdb.out => tst.format-cap-E.mdb.out} (100%)
 rename usr/src/cmd/mdb/test/format/{tst.format-G.mdb => tst.format-cap-G.mdb} (100%)
 rename usr/src/cmd/mdb/test/format/{tst.format-G.mdb.out => tst.format-cap-G.mdb.out} (100%)
 rename usr/src/cmd/mdb/test/format/{tst.format-J.mdb => tst.format-cap-J.mdb} (100%)
 rename usr/src/cmd/mdb/test/format/{tst.format-J.mdb.out => tst.format-cap-J.mdb.out} (100%)
 rename usr/src/cmd/mdb/test/format/{tst.format-P.mdb => tst.format-cap-P.mdb} (100%)
 rename usr/src/cmd/mdb/test/format/{tst.format-P.mdb.out => tst.format-cap-P.mdb.out} (100%)
 rename usr/src/cmd/mdb/test/format/{tst.format-R.mdb => tst.format-cap-R.mdb} (100%)
 rename usr/src/cmd/mdb/test/format/{tst.format-R.mdb.out => tst.format-cap-R.mdb.out} (100%)

diff --git a/usr/src/cmd/mdb/test/format/tst.format-E.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-E.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-E.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-E.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-E.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-E.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-E.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-E.mdb.out
diff --git a/usr/src/cmd/mdb/test/format/tst.format-G.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-G.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-G.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-G.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-G.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-G.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-G.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-G.mdb.out
diff --git a/usr/src/cmd/mdb/test/format/tst.format-J.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-J.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-J.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-J.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-J.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-J.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-J.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-J.mdb.out
diff --git a/usr/src/cmd/mdb/test/format/tst.format-P.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-P.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-P.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-P.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-P.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-P.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-P.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-P.mdb.out
diff --git a/usr/src/cmd/mdb/test/format/tst.format-R.mdb b/usr/src/cmd/mdb/test/format/tst.format-cap-R.mdb
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-R.mdb
rename to usr/src/cmd/mdb/test/format/tst.format-cap-R.mdb
diff --git a/usr/src/cmd/mdb/test/format/tst.format-R.mdb.out b/usr/src/cmd/mdb/test/format/tst.format-cap-R.mdb.out
similarity index 100%
rename from usr/src/cmd/mdb/test/format/tst.format-R.mdb.out
rename to usr/src/cmd/mdb/test/format/tst.format-cap-R.mdb.out
-- 
2.21.0

